common/db.go

```diff
@@ -46,7 +46,7 @@ type DBOrch struct {
 	PricePerPixel     int64
 	ActivationRound   int64
 	DeactivationRound int64
-	Stake             string
+	Stake             []byte
```

Comment:
I see very little to no benefit in changing this to be a blob, I also think strings are more ubiquitous here. 
Any specific reason for the change? 

I'd add that `BLOB` is never actually stored in the table, it's a pointer lookup so I think `STRING` is also more efficient here.

Reply:
> Any specific reason for the change?

The main reason was to avoid an additional check to determine if the string is successfully converted into a `big.Int`.

Ex.

```
val, ok := new(big.Int).SetString(stake)
if !ok {
    // Handle error here
}
```

vs.

```
val := new(big.Int).SetBytes(stake)
```

I includes these changes here mainly because I wanted to see what the scope would be.

> I also think strings are more ubiquitous here.

What do you mean strings are more ubiquitous? In the context of the Livepeer codebase or in the context of all codebases that need to store a `uint256` in a DB? If the former, its worth noting that we currently do use the BLOB type for the ticket [faceValue and winProb](https://github.com/livepeer/go-livepeer/blob/master/common/db.go#L105) which are `uint256` values. If the latter, could you point me to the right place to see `uint256` -> string conversions for DB storage?

One annoyance with using the `string` column type seems to be that it will be mapped to one of SQLite's storage classes so you have to consider how SQLite is going to coerce values - see the section on [type affinity](https://www.sqlite.org/datatype3.html).

Reply:
I don't think the coercing matters in this specific case when dealing with big.Ints converted to strings 

> A string might look like a floating-point literal with a decimal point and/or exponent notation but as long as the value can be expressed as an integer, the NUMERIC affinity will convert it into an integer. Hence, the string '3.0e+5' is stored in a column with NUMERIC affinity as the integer 300000, not as the floating point value 300000.0.

Anyway the performance between `TEXT` and `BLOB` does seem the same so I'm fine with using `BLOB` as it does avoid that additional check and having to validate nullstrings.